cloudify_types/cloudify_types/component/tests/test_blueprint_handling.py

```diff
@@ -128,3 +128,46 @@ def test_upload_blueprint_use_not_existing_external(self):
                 'Blueprint ID "test" does not exist'
             ):
                 upload_blueprint(**self.resource_config)
+
+    def test_upload_blueprint_bad_labels(self):
+        with mock.patch('cloudify.manager.get_rest_client') as mock_client:
+            blueprint_labels = {'foo': 'bar'}
```

Comment:
It's not exactly `blueprint_labels` (see the comment above).  It's just naming thing though, but might be confusing for someone looking at your code some time later.